crypto/dh/dh_group_params.c

```diff
@@ -139,9 +126,6 @@ static DH *dh_new_by_group_name(OPENSSL_CTX *libctx, const char *name)
 
     for (i = 0; i < (int)OSSL_NELEM(dh_named_groups); ++i) {
         if (strcasecmp(dh_named_groups[i].name, name) == 0) {
-            int max_target_security_strength =
-                ifc_ffc_compute_security_bits(dh_named_groups[i].nbits);
-
             /*
              * The last parameter specified here is
```

Comment:
This comment needs fixing with the last parameter being removed.

Reply:
Ahh yes - it was in my first pass before my machine shutdown and corrupted this file :)